content/docs/introduction/comparison.md

```diff
@@ -246,31 +246,38 @@ environment, then Prometheus is a good choice.
 
 ### Storage
 
-Sensu uses Redis to persist monitoring data, including the Sensu client registry, check results, check execution history, and current event data.
+Sensu stores current and recent event status information and real-time inventory data in an embedded database (etcd) or an external RDBMS (Postgres). 
 
 ### Architecture
 
-Sensu has a [number of components](https://docs.sensu.io/sensu-core/latest/overview/architecture/). It uses
-RabbitMQ as a transport, Redis for current state, and a separate server for
-processing and API access.
-
-All components of a Sensu deployment (RabbitMQ, Redis, and Sensu Server/API) can be clustered for highly available and redundant configurations.
+All components of a Sensu deployment can be clustered for high availability and improved event-processing throughput. Sensu also provides built-in support for [cluster federation](https://docs.sensu.io/sensu-go/latest/operations/deploy-sensu/use-federation/#configure-federation), including RBAC and other [API resource replication](https://docs.sensu.io/sensu-go/latest/operations/deploy-sensu/etcdreplicators/), which is recommended for deployments larger than 30,000 nodes under management.
 
 ### Summary
-If you have an existing Nagios setup that you wish to scale as-is, or want to take advantage of the automatic registration feature of Sensu, then Sensu is a good choice.
 
-If you want to do whitebox monitoring, or have a very dynamic or cloud based environment, then Prometheus is a good choice.
+Sensu and Prometheus have a few capabilities in common, but they take very different approaches to monitoring. Both offer extensible discovery mechanisms for dynamic cloud-based environments and ephemeral compute platforms, though the underlying mechanisms are quite different. Both provide support for collecting multi-dimensional metrics via labels and annotations. Both have extensive integrations, and Sensu natively supports collecting metrics from all Prometheus exporters. Both are capable of forwarding observability data to third-party data platforms (e.g. InfluxDB, Elasticsearch, Splunk). Where Sensu and Prometheus differ the most is in their use cases.
```

Comment:
I'm not sure it make sense to mention specific 3rd party things to forward to, as there's not going to be much overlap between ones where it'd make sense to send both types of data to I suspect.

Reply:
how about a generic description: "forwarding data to third-party data platforms (e.g. other event stores or TSDBs)". Neither platform is limited in terms of the different types of data platforms that data can be forwarded to, but a few categorical examples might help the reader to understand the common use cases.